### DISCUSSION TITLE
Increase number of values in filters web part

### DISCUSSION THREAD

**Discussion by PapaGeil** [microsoft-search/pnp-modern-search#3887] [Ideas]
Hi all,

The limit of 1000 values per filter is a bit frustrating for some of the filters we use. I'd like to suggest to further increase the number of values that can be handled, if possible.

Thank you

**Comment by kasperbolarsen** [comment]
Please see https://github.com/microsoft-search/pnp-modern-search/issues/2879  , seems to be an API limit and out of our hands

**Comment by PapaGeil** [comment]
Hi @kasperbolarsen,

Thank you for your swift reply! Too bad this apparently is a limitation that can't be fixed...
